build.gradle

```diff
@@ -2,17 +2,7 @@
 buildscript {
     // Gradle Plugins
     dependencies {
-        // Google Services - https://developers.google.com/android/guides/google-services-plugin
-        classpath "com.google.gms:google-services:4.3.14"
-        // Open source licenses plugin
-        classpath 'com.google.android.gms:oss-licenses-plugin:0.10.5'
-    }
-
-    // Force proguard to the new version to fix "proguard.ParseException: Use of generics not allowed for java type at '<1>_<2>_<3>JsonAdapter' in line 31 of file '/Users/philip/.gradle/caches/transforms-2/files-2.1/d5ca2d039e1d32e9ba6bfd6a67df5151/META-INF/proguard/moshi.pro'"
-    configurations.all {
-        resolutionStrategy {
-            force 'net.sf.proguard:proguard-gradle:6.1.1'
-        }
```

Comment:
I thought it was worth cleaning up this file while I was here. Proguard is up to 6.2.2 so I don't believe we need this anymore. I have tested the parsing of JSON in a release build with the user sync and everything seems fine.